ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -193,6 +193,7 @@ function useTemplateState(pendingConfirmation) {
 
 export default function ConfirmationPage({
   redirectToHomeOnZeroConfirmations = true,
+  hideSubmitButton = false,
```

Comment:
Does this need to be a component level property?

We'll only have one instance for the client so is it more explicit to always make this an approval template decision?

Reply:
Done.